backend-rust/src/graphql_api.rs

```diff
@@ -2343,11 +2350,27 @@ struct AccountAddressAmount {
     amount:          Amount,
 }
 
-#[derive(SimpleObject)]
+type AccountReleaseScheduleItemIndex = i64;
+
 struct AccountReleaseScheduleItem {
-    transaction: Transaction,
-    timestamp:   DateTime,
-    amount:      Amount,
+    index:             AccountReleaseScheduleItemIndex,
+    transaction_index: TransactionIndex,
+    timestamp:         DateTime,
+    amount:            Amount,
+}
+#[Object]
+impl AccountReleaseScheduleItem {
+    async fn transaction(&self, ctx: &Context<'_>) -> ApiResult<Transaction> {
+        Transaction::query_by_index(get_pool(ctx)?, self.transaction_index).await?.ok_or(
+            ApiError::InternalError(
+                "AccountReleaseScheduleItem: No transaction at transaction_index".to_string(),
+            ),
+        )
+    }
+
+    async fn timestamp(&self) -> DateTime { self.timestamp }
+
+    async fn amount(&self) -> Amount { self.amount }
 }
```

Comment:
It would expose the `index` and `transaction_index` as well in the API, so they would need to be skipped:
```
#[derive(SimpleObject)]
#[graphql(complex)]
struct AccountReleaseScheduleItem {
    #[graphql(skip)]
    index:             AccountReleaseScheduleItemIndex,
    #[graphql(skip)]
    transaction_index: TransactionIndex,
    timestamp:         DateTime,
    amount:            Amount,
}
```

But I prefer the current, mainly because it is more explicit and sort of isolate details about what goes into the API, especially since the `AccountReleaseScheduleItem` type is used to parse rows from the database.